### ISSUE TITLE
Patch needed to install Reflex? Dependency on haskell-src-exts-1.20.3

### ISSUE THREAD

**Issue by Zenleaf** [typelead/eta-hackage#142]
I get the following error when trying to build reflex for an Etlas project.
I am quite new to this, could you help me figure out if a patch is needed?
Here is part of the log:

Configuring library for haskell-src-exts-1.20.3..
Preprocessing library for haskell-src-exts-1.20.3..
unused rules: 777
Building library for haskell-src-exts-1.20.3..
[ 4 of 18] Compiling Language.Haskell.Exts.Syntaxeta: panic! (the 'impossible' happened)
  (Eta version 0.8.6b5):
	Error when generating class 'haskell_src_exts/language/haskell/exts/Syntax': putClassFile: ConstPool size exceeds 65,535 constants: 134672

Please report this as a Eta bug: http://github.com/typelead/eta/issues

etlas: Failed to build haskell-src-exts-1.20.3

**Comment by rahulmutt** [comment]
@Zenleaf Yes, `haskell-src-exts` requires a patch to split code into separate files until we find an automated way to fix the problem. This was introduced in recent versions of Eta to let you know at compile-time that something will crash at runtime.

**Comment by Zenleaf** [comment]
Awesome, thought so. Would like to give it a try when I have time.@rahulmutt
